tools/environment-setup.sh

```diff
@@ -42,4 +42,4 @@ lxc exec snapcraft-dev -- sudo -iu ubuntu bash -c \
     "echo 'source /home/ubuntu/.venv/snapcraft/bin/activate' >> .bashrc"
 
 echo "Container ready, enter it by running: "
-echo "lxc exec snapcraft-dev -- sudo -iu ubuntu bash"
+echo "lxc exec snapcraft-dev -- su - ubuntu"
```

Comment:
Thanks for the contribution. The man page for `su` states:

> It is recommended to always use the --login option (instead of its shortcut -) to avoid side effects caused by mixing environments.

Reply:
Oh sure, I prefer long options as well. I should have suggested `--login` initially.